app/server/pkg/community/google/analytics.go

```diff
@@ -105,6 +108,47 @@ func getTotalCounts() error {
 	return nil
 }
 
+//timeSeriesHelper is a helper function to add month-end dates to all monthly data
+func timeSeriesHelper(TimeData [][]string) ([][]string, error) {
```

Comment:
Is there any npm library for this? So that we don't have to make do this by ourselves

Reply:
This could have been done on frontend but the computation will unnecessarily increase whereas on the backend we are caching the data , also this same api will be required for Repute thus  have kept it in backend.

Reply:
ok